### PR TITLE
M0090IDE-4523 Update SDK 6 changelog for release 0.14.0 + update plugin version in examples

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**

### CODE REVIEW

File: SDK6UserGuide/sdk6ChangeLog.rst [hunk 1]
```diff
@@ -3,6 +3,43 @@
...
 
+.. _changelog-0.14.0:
+
+[0.14.0] - 2024-01-03
+~~~~~~~~~~~~~~~~~~~~~
+
+Added
+"""""
+
+- Add Jenkinsfile files to build and test with a JDK 17 and a JDK 21 (LTS versions).
```
Comment (denisbedin) [inline SDK6UserGuide/sdk6ChangeLog.rst:14]: If I remember correctly, we decided to not add Internal changes in the public changelog and only keep them in the changelog of the project.

File: SDK6UserGuide/howtos/buildExecutableWithMultipleVeePorts.rst [hunk 2]
```diff
@@ -41,7 +41,7 @@ For example, if you want to build an Executable for two VEE Ports, you can creat
   for example::
     
     plugins {
-        id("com.microej.gradle.application") version "0.13.0" apply false
+        id("com.microej.gradle.application") version "0.14.0" apply false
```
Comment (denisbedin) [inline SDK6UserGuide/howtos/buildExecutableWithMultipleVeePorts.rst:44]: The plugin's version must also be updated in the createProject and migrateMMMProject pages.

File: SDK6UserGuide/sdk6ChangeLog.rst (hunk 1 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/sdk6ChangeLog.rst:14]: Done. I forgot to save and commit them...

File: SDK6UserGuide/howtos/buildExecutableWithMultipleVeePorts.rst (hunk 2 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/howtos/buildExecutableWithMultipleVeePorts.rst:44]: Done. I forgot to save and commit them...

File: SDK6UserGuide/sdk6ChangeLog.rst (hunk 1 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/sdk6ChangeLog.rst:14]: Yes but I think it can be interesting to know that we test on other JDK versions. WDYT?

Comment (denisbedin) [reply SDK6UserGuide/sdk6ChangeLog.rst:14]: ok

**Review by denisbedin — APPROVED** [review]

**Pull request merged**